pyproject.toml

```diff
@@ -1,3 +1,65 @@
+[project]
+name = "pgcli"
+authors = [{ name = "Pgcli Core Team", email = "pgcli-dev@googlegroups.com" }]
+license = { text = "BSD" }
+description = "CLI for Postgres Database. With auto-completion and syntax highlighting."
+readme = "README.rst"
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: Unix",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: SQL",
+    "Topic :: Database",
+    "Topic :: Database :: Front-Ends",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+urls = { Homepage = "http://pgcli.com" }
+requires-python = ">=3.8"
+dependencies = [
+    "pgspecial>=2.0.0",
+    "click >= 4.1",
+    "Pygments>=2.0",
+    "prompt_toolkit>=2.0.6,<4.0.0",
+    "psycopg >= 3.0.14; sys_platform != 'win32'",
+    "psycopg-binary >= 3.0.14; sys_platform == 'win32'",
+    "sqlparse >=0.3.0,<0.6",
+    "configobj >= 5.0.6",
+    "cli_helpers[styles] >= 2.2.1",
+    "setproctitle >= 1.1.9",
```

Comment:
This requirement was conditional and there was a helpful comment that explained why. Both (the condition and the comment) should be retained.

Reply:
Sorry. Fixed.